### PR TITLE
[FEAT] Add ignore message filtering to syslog provider

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

File: configurations/aurora/scripts/make_aurora_config [hunk 1]
```diff
@@ -540,33 +540,42 @@ class MakeAuroraConfig(object):
...
+                        }
+                    },
+                    'HPCMLOG' : {
+                        'patternFilters' : {
+                            '.*No bootable device.*' : 'RasBootFailedNoBootableDevice',
+                            '.*Kernel panic.*' : 'RasProvKernelPanic',
+                            '.*cpu clock throttled.*' : 'RasProvNodeCpuThrottled',
+                            '.*Oops:.*' : 'RasProvNodeOops',
+                            '.*eeprom read at offset.*failed.*' : 'RasProvGigabyteEthernetEepromReadFailed',
+                            '.*mce:.*Machine Check:.*' : 'RasProvMachineCheckOccurred',
```
Comment (paul-amonson) [inline configurations/aurora/scripts/make_aurora_config:558]: Should this be moved to the aurora yml file?

**Review by paul-amonson — CHANGES_REQUESTED** [review]
Some related comments and I also don't see the changes to the various .yml files.

**Review by paul-amonson — APPROVED** [review]
Ok, I understand after a 1:1 and the extra needed work can be done in a separate review as this feature is functional.

File: configurations/eventsim/ProviderMonitoringNetworkForeignBus.json [hunk 2]
```diff
@@ -163,33 +163,42 @@
...
+        },
+        "HPCMLOG" : {
+          "patternFilters" : {
+            ".*No bootable device.*" : "RasBootFailedNoBootableDevice",
+            ".*Kernel panic.*" : "RasProvKernelPanic",
+            ".*cpu clock throttled.*" : "RasProvNodeCpuThrottled",
+            ".*Oops:.*" : "RasProvNodeOops",
+            ".*eeprom read at offset.*failed.*" : "RasProvGigabyteEthernetEepromReadFailed",
+            ".*mce:.*Machine Check:.*" : "RasProvMachineCheckOccurred",
+            ".*Machine Check Exception:.*" : "RasProvMachineCheckOccurred",
```
[the remark this answers is not in the document]
Comment (sindhu-106) [reply configurations/eventsim/ProviderMonitoringNetworkForeignBus.json:182]: These changes will be done in another review

**Review by yhuan47 — APPROVED** [review]

**Review by avincigu — APPROVED** [review]

**Pull request merged**